src/openforms/templates/includes/forms/field_wrapper.html

```diff
@@ -1,20 +1,25 @@
 {# Takes a 'field' context argument, which should be an instance of a BoundField #}
-<div class="openforms-form-control openforms-form-control--{{ type|default:'textfield' }}  utrecht-form-field">
```

Comment:
The removal of `utrecht-form-field` is intended?

Reply:
Yes, for regular fields it is included as a child, and radio/checkbox use fieldset rather than field